pages/profile/read/views/profile.jsx

```diff
@@ -0,0 +1,237 @@
+                            <Link page='project.list' label={project.title} />
+                          </p>
+                          <p>
+                            <span>
+                              <Snippet licenceNumber={project.licenceNumber}>
+                                projects.licenceNumber
+                              </Snippet>
+                            </span>
+                          </p>
+                          <p>
+                            <span>
+                              <Snippet
+                                expiryDate={format(
+                                  project.expiryDate,
+                                  dateFormat.medium
+                                )}
+                              >
+                                projects.expiryDate
+                              </Snippet>
+                            </span>
+                          </p>
+                        </Fragment>
+                      )
+                  )}
+                {isEmpty(activeProjects) && (
+                  <Snippet>projects.noProjects</Snippet>
+                )}
+              </dd>
+            </dl>
+          </Fragment>
+        )}
+
+        {
+          (allowedActions.includes('project.apply') && <Fragment>
+            <p>
+              <form method='POST' action={`/e/${estId}/projects/create`}>
+                <Button className='govuk-button add-margin'>
+                  <Snippet>buttons.pplApply</Snippet>
+                </Button>
+              </form>
+            </p>
+          </Fragment>)
+        }
+
+        <hr />
+
+        {
+          <Fragment>
+            <dl>
+              <dt>
+                <Snippet>responsibilities.title</Snippet>
+              </dt>
+
+              {!isEmpty(estRoles) && (
+                <Fragment>
+                  {estRoles.map(({ type, id }) => (
+                    <dd key={id}>{defineValue(type.toUpperCase())}</dd>
+                  ))}
+                </Fragment>
+              )}
+              {isEmpty(estRoles) && (
+                <dd>
+                  <Snippet>responsibilities.noRoles</Snippet>
+                </dd>
+              )}
+            </dl>
+          </Fragment>
+        }
+
+        <RoleApply
+          allowedActions={allowedActions}
+          estId={estId}
+          profileId={id}
+        />
+
+        <hr />
+
+        {
+          <dl>
+            <dt>
+              <Snippet>pil.title</Snippet>
+            </dt>
+            {pil && pil.licenceNumber && (
+              <Fragment>
+                <dd>
+                  <Link
+                    page='pil.read'
+                    pilId={pil.id}
+                    label={pil.licenceNumber}
+                  />
+                </dd>
+              </Fragment>
+            )}
+            {!pil && (
+              <dd>
+                <Snippet>pil.noPil</Snippet>
+              </dd>
+            )}
+          </dl>
+        }
```

Comment:
There's a random set of curly braces around the `dl`. Are they there for a reason?

Reply:
no reason - I have removed them